src/docs/implementation/SwateManual/Docs04-Ontology-Term-Search.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: docs
+title: Ontology Term Search
+published: 2022-09-19
+Author: Kevin Frey
+add toc: false
+add sidebar: _sidebars\swateManualSidebar.md
+---
+
+<br>
+
+Altough i will mostly talk about the Ontology term search on this page, it will also feature useful information about term search related to the *Building Blocks* page. Such general applicable information will be listed under [General](#General).
+
+Ontologies are controlled vocabularies of Terms. Most of these Terms feature a name, a unique identifier and a description. We use these Ontology Terms for workflow description to utilize the benefits of a unified language in terms of machine-readability and reproducibility, following [FAIR principles](https://www.go-fair.org/fair-principles/).
+
+- [General](#general)
+- [Term Insert](#term-insert)
+- [Relationship-Directed Search](#relationship-directed-search)
+  - [Relationship-Directed Search Theory](#relationship-directed-search-theory)
+  - [Relationship-Directed Search Usage](#relationship-directed-search-usage)
+
+# General
+
+You can find ontology term search fields on the *Building Blocks* and on the *Terms* tab in the <a href="./../../img/Swate-Overlay-Exp.jpg" target="_blank">navbar</a>.
+
+**Features:**
+- Access to the SwateDB with a list of established, [external ontologies](https://github.com/nfdi4plants/nfdi4plants_ontology/blob/main/ext_ontologies.include) deemed fit for use in plant science. In addition we feature our very own ontology [nfdi4pso](https://github.com/nfdi4plants/nfdi4plants_ontology/blob/main/nfdi4plants_ontology.obo), which we extend with missing but necessary terms.
+    - External ontologies are taken from the [obofoundry](https://obofoundry.org) and updated regularly.
+    - 👀 You can always ask us to add existing terms/ontologies or even new terms to our database. If you wish to do so, you can fill out an issue [here](https://github.com/nfdi4plants/nfdi4plants_ontology/issues/new/choose).
```

Comment:
This is not the recommended way to do this. We would prefer helpdesk and/or freetext input, which we should mention here